samples/package.json

```diff
@@ -12,7 +12,7 @@
   "license": "MIT",
   "dependencies": {
     "@pxtrn/bybit-api": "^1.1.5",
-    "binance": "^1.3.5",
+    "binance": "^2.8.9",
```

Comment:
Are you sure about this major version change in the binance sdk? Have you tested it with the other samples? I ask since I don't think v2 is an easy upgrade from v1, if I remember correctly...